db/models/interview.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const db = require('APP/db');
+
+const Interview = db.define('interviews', {
+  date: {
+    type: Sequelize.DATE,
+    allowNull: false
+  },
+  position: Sequelize.STRING,
+  status: Sequelize.ENUM('planned', 'done'),
+  candidateRating: {
```

Comment:
maybe we should name this `candidateOverallRating`? possibly to distinguish it from `candidateRating` from the join table. for our own sake, shouldn't matter in the execution